cloudpickle/cloudpickle.py

```diff
@@ -388,6 +388,11 @@ def _cell_set_factory(value):
 
 
 def _builtin_type(name):
+    if name == "ClassType":  # pragma: no cover
```

Comment:
I wonder if in this kind of situation we could
- be nice, provide a fix but emit a warning saying that this pickle has been created using an old version of `cloudpickle` and that this is not part of `cloudpickle`'s contract
- then, one or two minor versions later, we can safely suppress the fix.

Pretty much like a deprecation warning.

Reply:
The following reproduces the issue on my machine:
dump script (`cloudpickle 1.2.1`)
```python
In [3]: import cloudpickle
   ...: class A:
   ...:     pass
   ...: with open('pickle_file.pkl', 'wb') as f:
   ...:     cloudpickle.dump(A, f)
```
load script (`cloudpickle 1.4.0`)
```python
In [3]: import pickle
   ...: with open('pickle_file.pkl', 'rb') as f:
   ...:     A = pickle.load(f)
```

Reply:
Weird I thought I did this. Let me try again.

Reply:
Actually let's be nice and silent for now and we will add a `FutureWarning` in an upcoming release.

Reply:
I'm okay with this.